llvm/test/CodeGen/AArch64/replace-with-veclib-armpl.ll

```diff
@@ -15,7 +15,7 @@ declare <vscale x 2 x double> @llvm.cos.nxv2f64(<vscale x 2 x double>)
 declare <vscale x 4 x float> @llvm.cos.nxv4f32(<vscale x 4 x float>)
```

Comment:
I was expecting 
- `llvm/test/CodeGen/AArch64/replace-with-veclib-sleef.ll` 
- `llvm/test/CodeGen/AArch64/replace-with-veclib-sleef-scalable.ll`
- `llvm/test/CodeGen/AArch64/vec-libcalls.ll`
- `llvm/test/Transforms/LoopVectorize/AArch64/veclib-calls-libsystem-darwin.ll`
- `llvm/test/Transforms/LoopVectorize/AArch64/veclib-intrinsic-calls.ll`
I think you may have done some of these in another PR. But want to know what the plan is to do the rest?